package.json

```diff
@@ -1,63 +1,72 @@
-  "version": "2.1.0",
-  "description": "A verbose API library for Pterodactyl",
-  "author": "Devonte <https://github.com/devnote-dev>",
-  "contributors": [
-    "Chelog <https://github.com/chelog>",
-    "Cain <https://github.com/cainthebest>"
-  ],
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/PteroPackages/PteroJS/issues"
-  },
-  "homepage": "https://pteropackages.github.io/PteroJS/",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/PteroPackages/PteroJS.git"
-  },
-  "main": "./dist/src/index.js",
-  "types": "./dist/src/types/src",
-  "files": [
-    "dist",
-    "CHANGELOG.md",
-    "LICENSE",
-    "pterojs.json",
-    "README.md"
-  ],
-  "engines": {
-    "node": ">=14.16"
-  },
-  "scripts": {
-    "prepublish": "tsc",
-    "build": "tsc",
-    "docs": "typedoc --out docs --excludePrivate src/index.ts",
-    "test": "jest",
-    "test:watch": "jest --watchAll"
-  },
-  "keywords": [
-    "api",
-    "nodejs",
-    "library",
-    "wrapper",
-    "javascript",
-    "typescript",
-    "pterodactyl",
-    "pterodactyl-api"
-  ],
-  "dependencies": {
-    "axios": "^0.27.2",
-    "form-data": "^4.0.0",
-    "ws": "^8.8.1"
-  },
-  "devDependencies": {
-    "@types/jest": "^29.0.0",
-    "@types/node": "^17.0.45",
-    "@types/ws": "^8.5.3",
-    "jest": "^29.0.2",
-    "ts-jest": "^28.0.8",
-    "ts-node": "^10.9.1",
-    "typedoc": "^0.22.18",
-    "typescript": "^4.7.4"
-  }
-}
+{
+    "name": "@devnote-dev/pterojs",
+    "version": "2.1.0",
+    "description": "A verbose API library for Pterodactyl",
+    "author": "Devonte <https://github.com/devnote-dev>",
+    "contributors": [
+        "Chelog <https://github.com/chelog>",
+        "Cain <https://github.com/cainthebest>"
+    ],
+    "license": "MIT",
+    "bugs": {
+        "url": "https://github.com/PteroPackages/PteroJS/issues"
+    },
+    "homepage": "https://pteropackages.github.io/PteroJS/",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/PteroPackages/PteroJS.git"
+    },
+    "main": "./dist/index.js",
+    "types": "./dist/index.d.ts",
+    "files": [
+        "dist",
+        "CHANGELOG.md",
+        "LICENSE",
+        "pterojs.json",
+        "README.md"
+    ],
+    "engines": {
+        "node": ">=14.16"
+    },
+    "exports": {
+        ".": {
+            "require": "./dist/index.js",
+            "import": "./dist/index.mjs"
+        }
+    },
+    "scripts": {
+        "prepublish": "tsup-node",
+        "build": "tsup-node",
+        "format": "prettier --write **/**.{ts,json,md,yaml,yml}",
+        "docs": "typedoc --out docs --excludePrivate src/index.ts",
+        "test": "jest",
+        "test:watch": "jest --watchAll"
+    },
+    "keywords": [
+        "api",
+        "nodejs",
+        "library",
+        "wrapper",
+        "javascript",
+        "typescript",
+        "pterodactyl",
+        "pterodactyl-api"
+    ],
+    "dependencies": {
+        "axios": "^1.1.2",
```

Comment:
Please revert the package.json changes or downgrade Axios to its previous version, the latest isn't supported right now.

Reply:
axios reverted from `1.1.2` -> `0.27.2` in https://github.com/PteroPackages/PteroJS/pull/20/commits/3917b3f21cae71e42dc2cf7f09d7f9ab3af4dae2